collections/Rawl_B/MS_Rawl_B_489.xml

```diff
@@ -110,7 +110,7 @@
                      </recordHist>
                   </adminInfo>
                   <surrogates>
-                     <bibl type="digital-fascimile" subtype="full"><ref target="http://image.ox.ac.uk/list?collection=bodleian"><title>Early Manuscripts at Oxford</title></ref> <note>(full digital facsimile)</note></bibl>
+                     <bibl type="digital-fascimile" subtype="full"><ref target="https://digital.bodleian.ox.ac.uk/inquire/p/a5918f5a-2149-47bb-857e-0792bab8085a"><title>Digital Bodleian</title></ref> <note>(full digital facsimile)</note></bibl>
```

Comment:
Typo in facsimile here too.

Reply:
If this is a systematic typo, perhaps @andrew-morrison can automatically check and update the records that have this typo?

Reply:
The indexing scripts and XSL allow for either fascimile or facsimile.

Fixing them in the TEI could be done by a find and replace across all files in Oxygen.

Type values are not specified by TEI P5, but we could add a schematron rule.

Reply:
How many records in the TEI catalogues use 'fascimile'?

Reply:
Currently, 425 use fascimile and 1920 use facsimile (not counting the new Fihrist ones I did yesterday, which are all facsimile.)

I've just added a schematron rule, which will be included in the next release.

Reply:
I'll create an issue to update fascimile -> facsimile so we can move the discussion there.